sat/cli/bootsys/mgmt_power.py

```diff
@@ -428,10 +428,71 @@ def do_power_on_ncns(args):
                         except FatalPlatformError as err:
                             LOGGER.error(f'Failed to unfreeze Ceph on storage NCNs: {err}')
                             sys.exit(1)
-                        LOGGER.info('Ceph freeze completed successfully on storage NCNs.')
+                        LOGGER.info('Ceph unfreeze completed successfully on storage NCNs.')
+
+                        # Mount Ceph and S3FS filesystems on ncn-m001
+                        try:
+                            mount_filesystems_on_ncn('ncn-m001')
+                        except MountError as err:
+                            LOGGER.error(f'Failed to mount filesystems on ncn-m001: {err}')
+                            sys.exit(1)
+                        LOGGER.info('Mounted Ceph and S3FS filesystems on ncn-m001 successfully.')
 
         except ConsoleLoggingError as err:
             LOGGER.error(f'Aborting boot of NCNs due failure to set up NCN console logging: {err}')
             raise SystemExit(1)
 
     LOGGER.info('Succeeded with {}.'.format(action_msg))
+
+
+def mount_filesystems_on_ncn(ncn):
+    """Mount Ceph and S3FS filesystems on a given NCN.
+
+    Args:
+        ncn: The hostname of the NCN where the filesystems will be mounted.
+
+    Raises:
+        MountError: If mounting any filesystem fails.
+    """
+    filesystems = {
+        "ceph": "awk '{ if ($3 == \"ceph\") { print $2; }}' /etc/fstab",
+        "fuse.s3fs": "awk '{ if ($3 == \"fuse.s3fs\") { print $2; }}' /etc/fstab"
+    }
+
+    for fs_type, awk_command in filesystems.items():
+        try:
+            result = subprocess.run(awk_command, shell=True, capture_output=True, text=True, check=True)
+            mount_points = result.stdout.splitlines()
+
+            for mount_point in mount_points:
+                result = subprocess.run(["mountpoint", mount_point], capture_output=True, text=True)
```

Comment:
I would suggest adding more info-level log messages which can show what is being mounted. With these messages, you will be able to see whether the correct filesystems are identified in your testing output. For example:

```suggestion
                LOGGER.info(f'Checking whether {fs_type} filesystem is mounted on {mount_point}.')
                result = subprocess.run(["mountpoint", mount_point], capture_output=True, text=True)
```